### PR TITLE
bump_ocp_releases: added generate-bundle

### PR DESCRIPTION
**Pull request by danielerez**
Added missing 'make generate-bundle'

### CODE REVIEW

**Review by osherdp — APPROVED** [review]

File: Dockerfile.assisted-installer-deployment [hunk 1]
```diff
@@ -30,3 +26,8 @@ WORKDIR /home/assisted-installer-deployment
 COPY . .
 
 RUN pip install .
+
+# required for generating configuration/bundle in assisted-service repo
+FROM registry.ci.openshift.org/openshift/release:golang-1.16
+COPY ./setup_env.sh ./
+RUN ./setup_env.sh assisted_service
```
Comment (osherdp) [inline Dockerfile.assisted-installer-deployment:33]: this is probably problematic, as it's probably changes the current directory. no?

Comment (osherdp) [reply Dockerfile.assisted-installer-deployment:33]: why we need to be based on ``registry.ci.openshift.org/openshift/release:golang-1.16``?

**Review by osherdp — CHANGES_REQUESTED** [review]

File: Dockerfile.assisted-installer-deployment (hunk 1 above)
Comment (danielerez) [reply Dockerfile.assisted-installer-deployment]: no mandatory, added to dnf install instead

**Review by osherdp — APPROVED** [review]

**Pull request merged**